docs/ide/reference/how-to-use-the-keyboard-exclusively.md

```diff
@@ -65,17 +65,17 @@ To move among the options in a dialog box and change option settings by using on
 
 ## Navigate between windows and files
 
-- To move among files in an editor or designer, choose the **Ctrl**+**Tab** keyboard shortcut to display the IDE Navigator with **Active Files** selected. Choose the **Enter** key to navigate to the highlighted file.
+- To move among files in an editor or designer, use the **Ctrl**+**Tab** keyboard shortcut to display the IDE Navigator with **Active Files** selected. Use the **Enter** key to navigate to the highlighted file.
```

Comment:
Please change instances of "Use the **Enter** key" to "Press the **Enter** key." In other words, we use the verb "press" in an imperative statement regarding specific keys on a keyboard. (The first instance of "use the _x_ shortcut" is fine, however.) 

This is also per examples in the guidance link listed in the previous comment.